src/components/PrescriptionTr.tsx

```diff
@@ -72,14 +72,19 @@ const PrescriptionTr: FunctionComponent<PrescriptionTrProps> = ({ prescription,
                   />
                 </div>
               </div>
-              <hr className='my-2' />
-              <div className='d-flex align-items-center mb-3'>
-                <small><strong className='me-5'>{t('text.endPrescription')}</strong><Badge className='ms-5' pill bg={isValidIconClass}> {prescriptionEndDate}</Badge></small>
-
+              <div className='d-flex align-items-start '>
+                <small>
+                  <strong className='me-5'>
+                    {t('text.endPrescription')}
+                  </strong>
+                </small>
+                <Badge className='ms-4' pill bg={isValidIconClass}>
+                  {prescriptionEndDate}
+                </Badge>
               </div>
               <div className='d-flex align-items-center'>
                 <small><strong>{t('text.prescription')}:</strong></small>
-                <small>{prescription.photo_prescription === null ? <Button size='sm' className='ms-2' variant='primary' href={prescription.photo_prescription} disabled><FontAwesomeIcon icon={['fas', 'eye']} /></Button> : <Button size='sm' className='ms-2' variant='primary' href={prescription.photo_prescription} target='_blank'><FontAwesomeIcon icon={['fas', 'eye']} /></Button>}</small>
+                <small>{prescription.photo_prescription === null ? <Button size='sm' className='ms-2' variant='primary' href={prescription.photo_prescription} disabled><FontAwesomeIcon size='sm' icon={['fas', 'eye']} /></Button> : <Button size='sm' className='ms-2' variant='primary' href={prescription.photo_prescription} target='_blank'><FontAwesomeIcon size='sm' icon={['fas', 'eye']} /></Button>}</small>
```

Comment:
I know this is slightly out of scope, but can't we make the ternary operator to simply add "disabled" rather than recreating the components entirely in both cases?
Something like:
```tsx
<small>
  <Button size='sm' className='ms-2' variant='primary' href={prescription.photo_prescription} disabled={prescription.photo_prescription === null}>
    <FontAwesomeIcon size='sm' icon={['fas', 'eye']} />
  </Button>
</small>
```